src/Compilers/CSharp/Test/Symbol/Symbols/Source/FileModifierTests.cs

```diff
@@ -4022,4 +4083,419 @@ public void SyntaxTreeAlreadyPresent()
+
+            internal partial class Foo : IFoo
+            {
+                IReadOnlyDictionary<int, string> IFoo.Bar => _bar;
+            }
+            """;
+
+        var comp = CreateCompilation(new[] { (source1, "F1.cs"), (source2, "F2.cs") });
+        comp.VerifyDiagnostics();
+    }
+
+    [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/68219")]
+    public void PartialExplicitImplementation_04()
+    {
+        var source1 = """
+            file interface IFoo
+            {
+                int Bar { get; }
+            }
+
+            internal partial class Foo : IFoo
+            {
+            }
+            """;
+
+        var source2 = """
+            file interface IFoo
+            {
+                int Bar { get; }
+            }
+
+            internal partial class Foo : IFoo
+            {
+                int IFoo.Bar => 1;
+                int IFoo.Bar => 2;
+            }
+            """;
+
+        var comp = CreateCompilation(new[] { (source1, "F1.cs"), (source2, "F2.cs") });
+        comp.VerifyDiagnostics(
+            // F1.cs(6,24): error CS8646: 'IFoo.Bar' is explicitly implemented more than once.
+            // internal partial class Foo : IFoo
+            Diagnostic(ErrorCode.ERR_DuplicateExplicitImpl, "Foo").WithArguments("IFoo.Bar").WithLocation(6, 24),
+            // F1.cs(6,30): error CS0535: 'Foo' does not implement interface member 'IFoo.Bar'
+            // internal partial class Foo : IFoo
+            Diagnostic(ErrorCode.ERR_UnimplementedInterfaceMember, "IFoo").WithArguments("Foo", "IFoo.Bar").WithLocation(6, 30),
+            // F2.cs(9,14): error CS0102: The type 'Foo' already contains a definition for 'IFoo.Bar'
+            //     int IFoo.Bar => 2;
+            Diagnostic(ErrorCode.ERR_DuplicateNameInClass, "Bar").WithArguments("Foo", "IFoo.Bar").WithLocation(9, 14));
+    }
+
+    [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/68219")]
+    public void PartialExplicitImplementation_05()
+    {
+        var source0 = """
+            var c = new C();
+            c.Use1();
+            c.Use2();
+            """;
+
+        var source1 = """
+            using System;
+
+            file interface FI
+            {
+                int Bar { get; }
+            }
+
+            internal partial class C : FI
+            {
+                int FI.Bar => 1;
+                public void Use1() => Console.Write(((FI)this).Bar);
+            }
+            """;
+
+        var source2 = """
+            using System;
+
+            file interface FI
+            {
+                int Bar { get; }
+            }
+
+            internal partial class C : FI
+            {
+                int FI.Bar => 2;
+                public void Use2() => Console.Write(((FI)this).Bar);
+            }
+            """;
+
+        var verifier = CompileAndVerify(new[] { (source0, "F0.cs"), (source1, "F1.cs"), (source2, "F2.cs") }, verify: Verification.Skipped, expectedOutput: "12");
```

Comment:
>verify: Verification.Skipped [](http://example.com/codeflow?start=110&length=28)

Why is IL verification skipped in the newly added tests (but none of the previous file-type tests did)?

Reply:
ILVerify incorrectly believes duplicate members are present in the tests where the code is executed.

Reply:
Could you produce an IL/ildasm example for this? How is it possible given our current naming convention for file-types?
If we can confirm this is an ILVerify issue, then we should file an issue on the runtime repo.

Reply:
I think the issue is the name+parameter list of each member is exactly the same.

Perhaps somewhere along the way the member should be adjusted to prefix with the metadata name of the file type, instead of its source name.